handlers/challenge_handler.py

```diff
@@ -500,7 +508,10 @@ def init(self, slack_wrapper):
                     ctf.add_challenge(challenge)
 
         # Create the database accordingly
-        pickle.dump(database, open(self.DB, "wb+"))
+        pickle.dump(database, open(ChallengeHandler.DB, "wb+"))
+
+    def init(self, slack_wrapper):
+        ChallengeHandler.update_database_from_slack(slack_wrapper)
```

Comment:
`init` is a proper method. So this could use `self.update_database_from_slack`.

We should change the registration of handlers so that we do NOT instantiate the handler class, in which case we'd change all of these to class or static methods.

I'll file a separate ticket.